test/integrations/Callouts.test.js

```diff
@@ -0,0 +1,37 @@
+import 'babel-polyfill';
+import integration from '../helpers/integration';
+
+const { it, isIOS } = integration;
+
+describe('Callouts', () => {
+  // For some reason appium can't see if the marker is opened or not
+  it('should show custom callouts', function* callback(driver, done) {
+    yield driver.elementById('Custom Callouts').click();
+
+    yield driver.elementById('Tap on markers to see different callouts');
```

Comment:
What does this do, just calling `elementById()` but not clicking or anything? It waits for it?